ros/content.md

```diff
@@ -1,50 +1,158 @@
-# What is [ROS](http://www.ros.org/)?
+# What is [ROS](https://www.ros.org/)?
 
-The Robot Operating System (ROS) is a set of software libraries and tools that help you build robot applications. From drivers to state-of-the-art algorithms, and with powerful developer tools, ROS has what you need for your next robotics project. And it's all open source.
+The Robot Operating System (ROS) is a set of software libraries and tools that help you build robot applications.  
+From drivers to state-of-the-art algorithms, and with powerful developer tools, ROS has what you need for your next robotics project.  
+And it's all open source.
```

Comment:
I think you probably want to take a look at the "rich diff" in GitHub for this file -- using one-sentence-per-line results in literal line breaks in the output on GitHub (and Docker Hub's rendering is less consistent, which is why we tend to use one-line-per-paragraph even though it makes the diffs kind of annoying).

Reply:
I've reverted the one sentence per line commit.